docs/src/getting-started.md

```diff
@@ -9,7 +9,7 @@ Kani uses proof harnesses to analyze programs. Proof harnesses are similar to te
 
 ## Project Status
 
-Kani is currently under active development and has not made an official release yet.
+Kani is currently under active development. Releases are published [here](https://github.com/model-checking/kani/releases).
```

Comment:
We might want to point towards the install guide too? People might be lead astray by trying to download a release or something from this page.